src/Api.js

```diff
@@ -178,13 +178,31 @@ class Api {
 		this.resultsPromise = $.getJSON( this.getAjaxURL( wikiUrl ) )
 			.then( function ( result ) {
 				// Handle error response.
-				if ( !result.success && result.info ) {
+				if ( !result.success ) {
 					// The API gives us an error message, but we don't use it because it's only
-					// in English.
-					return $.Deferred().reject( 'ext-whowrotethat-api-error-wikiwho-generic' );
+					// in English. Some of the error messages are:
+					// * result.info: "Requested data is not currently available in WikiWho
+					//   database. It will be available soon."
+					// * result.error: "The article (x) you are trying to request does not exist in
+					//   english Wikipedia."
+					// We do add the full error details to the console, for easier debugging.
+					let errCode = 'contact';
```

Comment:
Pedantic comment: You could do

```
const errCode = ( result.info && result.info.match( /data is not currently available/i ) ?
     'refresh' : 'contact';
```

Reply:
I was thinking that we will want to extend this with matching different error messages, but you're right, it could be simpler for the time being.

Done.